zipline/data/us_equity_loader.py

```diff
@@ -126,6 +127,13 @@ def _get_adjustments_in_range(self, asset, dts, field):
             The days for which adjustment data is needed.
         field : str
             OHLCV field for which to get the adjustments.
+        is_perspective_after : bool
+            see: _ensure_sliding_window
+            If True, the index at which the Mulitply object should be popped
```

Comment:
some grammatical and spelling issues here, maybe from stitching together different versions of the docstring. 

maybe remove `the value before the time at which the adjustment is marked` ?  and clean up the typos.